behat-tests/README.txt

```diff
@@ -9,43 +9,44 @@ To set up your environment to run tests locally:
 1.  Download the Selenium stand-alone server JAR file:
     http://selenium-release.storage.googleapis.com/2.44/selenium-server-standalone-2.44.0.jar
 
-2.  Install Composer:
+2.  Selenium version 2.44 is only compatible with Firefox 33, but not 34 or above. You can download
```

Comment:
I'm using Firefox 34.  The bug is specific to Firefox version 35+ I believe.  Did you try downloading the latest selenium builds and trying it out - I see talk that they are a possibility and that a new version is also coming out.  I would still reference the bug conversations as there is useful information there to help people identify problems.